src/index.html

```diff
@@ -11,8 +11,56 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
Browsers do not directly interpret SCSS files. Ensure that the SCSS file is compiled to a CSS file and the correct CSS file path is provided here.